dtfs-central-api/api-tests/helpers/with-validate-audit-details.api-tests.ts

```diff
@@ -28,7 +28,13 @@ export const withValidateAuditDetailsTests = ({ makeRequest, validUserTypes = AU
     it('should return 400 if no auditDetails provided', async () => {
       const { status, body } = await makeRequest();
 
-      expect(body).toEqual({ status: 400, message: 'Invalid auditDetails, Missing property `userType`' });
+      // Error body can either come from payload validation or controller validation
+      if (Array.isArray(body)) {
+        expect(body).toEqual(['auditDetails: Invalid input (invalid_union)']);
```

Comment:
do you think we should look to return the same thing for both, as they are the same errors

Reply:
Technically we could, but the way to do that would involve overwriting the existing logic around how we validate audit details as we don't have loads of control over the return type of the zod validator. I will try to make it consistent across services though

Reply:
Thank you -- Even if we just try/catch the zod errors and throw the other error it might be better

Reply:
try-catching the zod error won't fix this because it's to do with how I format the error. I also think the `"Invalid auditDetails, Missing property 'userType'"` is misleading because the validation can fail for different reasons. Like I say, I'll try to come up with something that works